common.ini.in

```diff
@@ -26,3 +26,8 @@ discourse.url = {discourse_url}
 discourse.public_url = {discourse_url}
 discourse.api_key = {discourse_api_key}
 discourse.sso_secret = {discourse_sso_secret}
+
+mail.from = {mail_from}
+mail.host = {mail_host}
+mail.port = {mail_port}
+mail.debug = {mail_debug}
```

Comment:
We can add it easily afterwards when/if we use an smtp server with authentication.
Currently in v5 we are using the psemail server.